quad_utils/launch/test_ekf.launch

```diff
@@ -0,0 +1,45 @@
+<launch>
+  
+    <include file="$(find quad_utils)/launch/load_params.launch" />
+
+    <arg name="live_plot" default="false"/>
+
+    <group ns="/trajectory">
+        <!-- Launch the nodes to publish all the robot tfs and visualize in rviz -->
+        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
+            <remap from="joint_states" to="/visualization/joint_states/trajectory"/>
+            <param name="tf_prefix" value="/trajectory"/>
+        </node>
+    </group>
+
+    <group ns="/ground_truth">
+        <!-- Launch the nodes to publish all the robot tfs and visualize in rviz -->
+        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
+            <remap from="joint_states" to="/visualization/joint_states/ground_truth"/>
+            <param name="tf_prefix" value="ground_truth"/>
+        </node>
+    </group>
+
+    <group ns="/estimate">
+        <!-- Launch the node to publish state estimate messages-->
+        <node name="ekf_estimator" pkg="ekf_estimator" type="ekf_estimator_node" output="screen"/>
```

Comment:
This is likely why you needed to add the "/" in front of "robot_description" - if launching ekf estimator under a namespace, the resulting node would be /estimate/ekf_estimator, and looking for a parameter named "robot_description" would result in looking for "/estimate/robot_description". The leading "/" clarifies to look in the global namespace.

I think the best fix for this is to just launch the estimator outside of this namespace (either directly or ideally by including the estimation.launch file).